### PR TITLE
[Docs] Fix relative paths to code

### PR DESCRIPTION
**Pull request by stefanalund**
#### :heavy_check_mark: Checklist

- [ ] All tests are passing `yarn test`
- [ ] Screenshots attached (for UI changes)
- [ ] Relevant documentation updated
- [ ] Prettier run on changed files
- [ ] Tests added for new functionality
- [ ] Regression tests added for bug fixes

### CODE REVIEW

File: plugins/proxy-backend/README.md [hunk 1]
```diff
@@ -14,13 +14,13 @@ To run it within the backend do:
 
 1. Register the router in `packages/backend/src/index.ts`:
 
-```
+```tsx
```
Comment (benjdlambert) [inline plugins/proxy-backend/README.md:17]: just `ts` is fine here.

Comment (stefanalund) [reply plugins/proxy-backend/README.md:17]: It's used elsewhere in the docs, should we change all of them? @benjdlambert

Comment (benjdlambert) [reply plugins/proxy-backend/README.md:17]: I mean it really doesn't matter to be honest, it's just syntax highlighting. .tsx is when you normally have some React thrown in the mix, which isn't gonna happen in backend stuff. And I'm pretty sure that the colours are gonna be the same either way hah

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -54,6 +54,10 @@ If you are proposing a feature:
 - Remember that this is a volunteer-driven project, and that contributions
   are welcome :)
 
+## Add you company to ADOPTERS
```
Comment (benjdlambert) [inline CONTRIBUTING.md:57]: Your

**Review by benjdlambert — APPROVED** [review]

File: plugins/proxy-backend/README.md (hunk 1 above)
Comment (stefanalund) [reply plugins/proxy-backend/README.md:17]: Good point. Fixing.

**Pull request merged**